lib/benchee/suite.ex

```diff
@@ -47,3 +47,93 @@ defimpl DeepMerge.Resolver, for: Benchee.Suite do
     Map.merge(original, override, resolver)
   end
 end
+
+if Code.ensure_loaded?(Table.Reader) do
+  defimpl Table.Reader, for: Benchee.Suite do
+    alias Benchee.CollectionData
+    alias Benchee.Scenario
+
+    def init(suite_results) do
+      columns = get_columns_from_suite(suite_results)
```

Comment:
In the code base we usually refer to this as `suite` or well just the suite, so I think that's fine :thinking: